ldb/templates/ldb/base.html

```diff
@@ -1,12 +1,14 @@
 {% extends "base_bootstrap3.html" %}
+{% load i18n %}
 
 {% block menu %}
-  <li><a href="{% url 'ldb_index' %}">Zoeken</a></li>
-  <li><a href="{% url 'ldb_people_create' %}">Persoon toevoegen</a></li>
-  <li><a href="{% url 'ldb_organizations_create' %}">Organisatie toevoegen</a></li>
-  <li><a href="{% url 'ldb_committees' %}">Commissies</a></li>
-  <li><a href="{% url 'ldb_export' %}">Exporteren</a></li>
+  <li><a href="{% url 'ldb_index' %}">{% trans 'Search' %}</a></li>
+  <li><a href="{% url 'ldb_people_create' %}">{% trans 'Add person' %}</a></li>
   {% if user.is_staff %}
-    <li><a href="{% url 'admin:ldb_person_changelist' %}">Beheer</a></li>
+    <li><a href="{% url 'ldb_organizations_create' %}">{% trans 'Add organization' %}</a></li>
```

Comment:
Why are these links moved to the `is_staff` part, while the actual authorisation check is not updated?

Reply:
I thought that was the case due to [this comment](https://github.com/WISVCH/dienst2/pull/240#discussion_r297848498). Should this be changed somewhere else?

Reply:
That comment was about moving `admin:ldb_person_import` to the `is_staff` block, but I'm not sure why this PR now also moves `ldb_committees` and others.

Reply:
Only the `admin:...` urls should be in the `is_staff`